src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
```

Comment:
You already have a class named `Animal`, so to avoid confusion, it is better to name the function argument something else: aim, victim, target etc.

Reply:
> You already have a class named `Animal`, so to avoid confusion, it is better to name the function argument something else: aim, victim, target etc.

i can't use 'static' because of eslint, now i failed the test but i fixed it